buckw

```diff
@@ -59,18 +59,14 @@ jsonq() {
 }
 
 ensure python 'https://www.python.org'
-ensure ant 'http://ant.apache.org/'
 INSTALLED_WATCHMAN=`command -v watchman`
 
-DEFAULT_BUCK_REPO="https://github.com/facebook/buck.git"
-DEFAULT_BUCK_INSTALL_DIR="$HOME/.gradle/caches/okbuck/buck"
```

Comment:
Do we want to provide some sort of cleanup? This leaves the old repo checked out.

Reply:
I thought about it, but I did not want to force folks to reclone the buck repo if they were building an older sha for some reason.

That should be upto the users (who can modify their buckw to do the cleanup if they like)